src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.PropertiesRegistry.cs

```diff
@@ -12,14 +12,15 @@
 using System.Threading;
 using Uno.Collections;
 using Uno.UI.Helpers;
+using System.Collections;
 
 namespace Windows.UI.Xaml
 {
 	public sealed partial class DependencyProperty
 	{
 		private class DependencyPropertyRegistry
```

Comment:
This dictionary has a single static instance that is kept for the lifetime of the whole app.